src/app/forms/templates/hgv/hgv-tech-record.template.ts

```diff
@@ -0,0 +1,153 @@
+import { FormNode, FormNodeTypes, FormNodeViewTypes } from '../../services/dynamic-form.service';
+
+export const HgvTechRecord: FormNode = {
+    name: 'techRecordSummary',
+    label: 'Summary',
+    type: FormNodeTypes.GROUP,
+    children: [
+        {
+            name: 'vehicleType',
+            label: 'Vehicle type',
+            value: '',
+            children: [],
+            type: FormNodeTypes.CONTROL,
+            viewType: FormNodeViewTypes.VEHICLETYPE
+        },
+        {
+            name: 'regnDate',
+            label: 'Date of first registration',
+            value: '',
+            children: [],
+            type: FormNodeTypes.CONTROL,
+            viewType: FormNodeViewTypes.STRING
```

Comment:
With the default type above all of these string types can be removed

Reply:
Sorry, what do you mean? I am confused

Reply:
My above comment about the default switch on the view item list component. It means if you don't supply the view type it will default to String

Reply:
I'm okay if we leave them in and be explicit